scripts/vendor.sh

```diff
@@ -1,7 +1,10 @@
 #!/bin/bash
+
+set -ex
+
 mkdir -p vendor
 curl -o sqlite-amalgamation.zip https://www.sqlite.org/2024/sqlite-amalgamation-3450300.zip
-unzip -d
+# unzip -d
 unzip sqlite-amalgamation.zip
```

Comment:
Not sure if this commands are correct, this floating unzip -d was incorrect and I had to comment it so the amalgamation was correctly downloaded.